dev-db/sqlitestudio/sqlitestudio-3.2.1.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="de es fr it pl pt_BR ro_RO ru sk zh_CN"
+
+inherit eutils l10n qmake-utils xdg
+
+DESCRIPTION="Powerful cross-platform SQLite database manager"
+HOMEPAGE="https://sqlitestudio.pl"
+SRC_URI="https://sqlitestudio.pl/files/sqlitestudio3/complete/tar/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="cli cups tcl test"
+
+REQUIRED_USE="test? ( cli )"
+
+RDEPEND="
+	dev-db/sqlite:3
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtscript:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	cli? ( sys-libs/readline:* )
+	cups? ( dev-qt/qtprintsupport:5 )
+	tcl? ( dev-lang/tcl:* )
```

Comment:
this is almost certainly wrong, should be :0= to trigger rebuild on tcl subslot bump.